docs/migrating/from-gatsby.md

```diff
@@ -0,0 +1,235 @@
+---
+description: Learn how to transition an existing Gatsby project to Next.js.
+---
+
+# Migrating from Gatsby
+
+This guide will help you understand how to transition from an existing Gatsby project to Next.js. Since both frameworks are built on top of React, the transition can be broken down into a series of steps. Migrating to Next.js will allow you to:
+
+- Choose which [data fetching](/docs/basic-features/data-fetching) strategy you want on a per-page basis (instead of only SSG)
+- Use [Incremental Static Regeneration](/docs/basic-features/data-fetching#incremental-static-regeneration) to update _existing_ pages by re-rendering them in the background as traffic comes in
+- Use [API Routes](/docs/api-routes/introduction)
+
+And more! First, let's talk about running a Next.js application.
+
```

Comment:
Also, I strongly recommend linking to https://github.com/leerob/gatsby-to-nextjs/pull/1 as early as possible in this document. I think this PR is very helpful but no one will notice it if you put it at the end.

Reply:
I liked the suggestions about including diffs early – I've added those 👍